src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.NET.CrossGen.targets

```diff
@@ -131,6 +131,9 @@ Copyright (c) .NET Foundation. All rights reserved.
 
     <Message Text="CrossgenCommandline: $(CrossgenCommandline)"/>
 
+    <!-- Workaround for https://github.com/NuGet/Home/issues/4424 -->
+    <Exec Command="chmod 755 $(CrossgenExe)" Condition="'$(OS)' != 'Windows_NT'"/>
```

Comment:
Can we log a tracking bug to take this out when the underlying issue is fixed?  That way we remember to remove it.

In general, this is bad in our product because we don't know if we have access to do this or not.